src/sezzle-checkout-button/sezzle-checkout-button-asset.js

```diff
@@ -94,6 +98,51 @@ class SezzleCheckoutButton {
 	}
 
 	init() {
-		this.createButton()
+		try{
+			this.createButton()
+			this.eventLogger.sendEvent("checkout-button-onload")
+		}catch(e){
+			this.eventLogger.sendEvent("checkout-button-error", e.message)
+		}
 	}
 }
+
+class EventLogger {
+	constructor(options){
+		this.merchantUUID = options.merchantUUID || '';
+		this.widgetServerEventLogEndPoint = options.widgetServerEventLogEndPoint || 'https://widget.sezzle.com/v1/event/log';
```

Comment:
Tiny thing: I'd suggest getting only the domain name from SFS and concatenate the rest of the path here. In this case if we ever change the path (e.g. from v1 to v2), we will have to update it here only, and not in SFS (or SG at that time).

Reply:
addressed